includes/class-freemius.php

```diff
@@ -227,6 +227,9 @@ class Freemius extends Freemius_Abstract {
 		 */
 		private static $_instances = array();
 
+		// Module types
+		const MODULE_TYPE_PLUGIN = 'plugin';
```

Comment:
@fajardoleo I'm not 100% sure, but I think that class `const` is not supported in PHP 5.2. If it does please comment here, so I'll know as well. If it doesn't, please change that to defines.